src/generators/05_arbitrary_teleport.py

```diff
@@ -0,0 +1,81 @@
+from src.generators._base import BaseGenerator, GeneratorResult
+
+CREATE_TRIGGER = "scoreboard objectives add $trigger trigger"
+CREATE_SCOREBOARD = "scoreboard objectives add arbitary_location_$axis_$id dummy"
+SET_SCOREBOARD = "scoreboard players set @a arbitary_location_$axis_$id $val"
+SET_TP = """
+execute as @a[scores={set_tp_$id=1..}] in minecraft:overworld store result score @s arbitary_location_x_$id run $x
+execute as @a[scores={set_tp_$id=1..}] in minecraft:overworld store result score @s arbitary_location_y_$id run $y
+execute as @a[scores={set_tp_$id=1..}] in minecraft:overworld store result score @s arbitary_location_z_$id run $z
+scoreboard players set @a set_tp_$id 0
+scoreboard players enable @a set_tp_$id
+""".strip()
+
+TELEPORT_FLOW = """
+tellraw @a[scores={gurkoins=$price..,shop_tp_$id=1..}] ["",{"text":"Thank you for your purchase!","bold":true},{"text":" $price GRK have been subtracted from your account"}]
+tellraw @a[scores={gurkoins=..$infprice,shop_tp_$id=1..}] ["",{"text":"Not enough funds!","bold":true,"color":"red"},{"text":" $price GRK are needed to get buy that item"}]
+execute as @a[scores={gurkoins=$price..,shop_tp_$id=1..}] in minecraft:overworld $x
+execute as @a[scores={gurkoins=$price..,shop_tp_$id=1..}] in minecraft:overworld $y
+execute as @a[scores={gurkoins=$price..,shop_tp_$id=1..}] in minecraft:overworld $z
+scoreboard players set @a[scores={gurkoins=$price..,shop_tp_$id=1..}] lib_teleport_is_absolute 1
+execute as @a[scores={gurkoins=$price..,shop_tp_$id=1..}] in minecraft:overworld run function generated:lib_teleport/teleport
+scoreboard players remove @a[scores={gurkoins=$price..,shop_tp_$id=1..},gamemode=!creative] gurkoins $price
+scoreboard players set @a shop_tp_$id 0
+scoreboard players enable @a shop_tp_$id
+""".strip()
+
+class Generator(BaseGenerator):
+    name = "Generate arbitary teleports"
+
+    def generate(self) -> GeneratorResult:
+        folder = self.FUNCTIONS / "arbitary_teleport"
+        folder.mkdir(parents=True, exist_ok=True)
+
+        teleports = []
+        commands = []
+        setups = []
+
+        # home

```

Comment:
```suggestion
        # TP to home
```
Something like this?